Jellyfin.Plugin.Douban/FrodoUtils.cs

```diff
@@ -0,0 +1,180 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Security.Cryptography;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Web;
+using Jellyfin.Plugin.Douban.Response;
+using MediaBrowser.Controller.Entities;
+using MediaBrowser.Model.Entities;
+
+namespace Jellyfin.Plugin.Douban
+{
+    /// <summary>
+    /// Frodo is the secondary domain of API used by Douban APP.
+    /// </summary>
+    public class FrodoUtils
```

Comment:
nit: Maybe we can treat this as the douban frodo client, then we can encapsulate more related logic here. WDYT?

Reply:
I think the next step is to package the frodo code into lib, same like [TMDbLib](https://github.com/LordMike/TMDbLib).